spec/factories/move.rb

```diff
@@ -13,10 +13,6 @@
     sequence(:created_at) { |n| Time.now - n.minutes }
     sequence(:date_from) { |n| Date.today - n.days }
 
-    trait :requested do
-      status { 'requested' }
-    end
-
```

Comment:
This is redundant as the factory already specifies a default status of `requested`. There's also a trait named `requested` further down the same factory. Tests pass without this so I think it's safe to remove and cleans up the factory although isn't entirely related to this PR.